python/GafferSceneUI/CameraUI.py

```diff
@@ -39,11 +39,61 @@
 
 import imath
 
+import IECoreScene
+
 import Gaffer
 import GafferUI
 import GafferScene
 import GafferSceneUI
 
+filmFitModeMetadata = [
+
+	"description",
+	"""
+	Determines how the image resolution is scaled to
+	fit the camera aperture. If the aspect ratios of
+	the aperture and the output resolution are the
+	same, then this has no effect, otherwise it
+	dictates what method is used to preserve the pixel
+	aspect ratio of the rendered image.
+
+	Horizontal
+	:   The image is scaled so that it fills the full
+		width of the aperture and aspect ratio
+		is preserved.
+
+	Vertical
+	:   The image is scaled so that it fills the full
+		height of the aperture and aspect ratio
+		is preserved.
+
+	Fit
```

Comment:
I've got the wrong end of the stick obviously. I've always thought of film fit as fitting the image into the aperture, and the Resize as fitting the input format into the output format. But for the fill/fit modes to correlate between the film fit and resize, it seems it's better to think of fitting the aperture to the image?

Reply:
Wait a minute, I just played with this, and it's already working consistently?

For a camera: Fit (input)< aperture > into (output)< renderResolution >
For a resize: Fit (input)< input resolution > into (output)< output resolution >

If I take a camera with a square aperture which exactly matches a sphere, choose Fit, and render 1080P, I see the sphere fits vertically, with space at the sides.
If I make a square image with a circle that matches the image size, resize it using Fit to 1080P, I see the circle fits vertically, with space at the sides.

In both cases, choosing Fill would fit Horizontally, with the top/bottom of the circle outside the display window.

I was all set to argue that we should be matching conventions for cameras, rather than matching Resize, but it looks like everything is consistent anyway.

Reply:
> For a camera: Fit (input)< aperture > into (output)< renderResolution >

I think this was the root of my misunderstanding - I was thinking of things as fitting the resolution into the aperture. The help text does seem to encourage that misconception a little, so perhaps we can reword it along the lines of "how the aperture is adjusted to account for the render resolution?".